applications/tari_base_node/src/builder.rs

```diff
@@ -495,43 +493,155 @@ where
+                                    );
+                                    e
+                                });
+                            trace!(target: LOG_TARGET, "Attempting UTXO validation for Invalid Outputs.",);
+                        },
+                        ValidationTask::UnspentOutputs => {
+                            let _ = oms_handle
+                                .clone()
+                                .validate_utxos(UtxoValidationRetry::UntilSuccess)
+                                .await
+                                .map_err(|e| {
+                                    error!(
+                                        target: LOG_TARGET,
+                                        "Problem starting UTXO validation protocols in the Output Manager Service: {}",
+                                        e
+                                    );
+                                    e
+                                });
+                            trace!(target: LOG_TARGET, "Attempting UTXO validation for Unspent Outputs.",);
+                        },
+                    };
+                    loop {
+                        futures::select! {
+                            event = event_stream.select_next_some() => {
+                                match event.unwrap() {
+                                    // Restart the protocol if aborted (due to 'BaseNodeNotSynced')
+                                    OutputManagerEvent::UtxoValidationAborted(s) => {
+                                        trace!(
+                                            target: LOG_TARGET,
+                                            "UTXO validation event 'UtxoValidationAborted' ({}), restarting.", s,
+                                        );
+                                        break;
+                                    },
+                                    // Restart the protocol if failure
+                                    OutputManagerEvent::UtxoValidationFailure(s) => {
+                                        trace!(
+                                            target: LOG_TARGET,
+                                            "UTXO validation event 'UtxoValidationFailure' ({}), restarting.", s,
+                                        );
+                                        break;
+                                    },
+                                    // Exit upon success
+                                    OutputManagerEvent::UtxoValidationSuccess(s) => {
+                                        trace!(
+                                            target: LOG_TARGET,
+                                            "UTXO validation event 'UtxoValidationSuccess' ({}).", s,
+                                        );
+                                        if validation == ValidationTask::InvalidOutputs {
```

Comment:
I don't think you need to do this. You only care able the Unspent outputs validation.

Reply:
See comment above